packages/components/psammead-caption/src/index.jsx

```diff
@@ -38,4 +41,19 @@ const Caption = styled.figcaption`
   }
 `;
 
+const Caption = ({ children, script }) => {
+  const GEL_LONG_PRIMER = getLongPrimer(script);
+
+  return <StyledElement typography={GEL_LONG_PRIMER}>{children}</StyledElement>;
```

Comment:
```suggestion
  return <StyledCaption typography={GEL_LONG_PRIMER}>{children}</StyledCaption>;
```

Reply:
@pjlee11 Shall we apply the same naming `StyledComponentName` to the rest of components?